Documentation/ceph-filesystem.md

```diff
@@ -46,7 +46,7 @@ spec:
   dataPools:
     - replicated:
         size: 3
-  preservePoolsOnDelete: true
+  preserveFsOnDelete: true
```

Comment:
If `preserveFsOnDelete: true`, we will effectively preserve both the filesystem and pools, right? In other words, `preservePoolsOnDelete` is ignored if the former is true.

Reply:
And if it doesn't make sense to preserve pools without preserving the filesystem, how about adding validation that preservePoolsOnDelete isn't true when preserveFsOnDelete is false?